style.css

```diff
@@ -1 +1,172 @@
 /*  Add your styling here */
+body {
+  font: 1em sans-serif;
+  align-content: center;
+  align-items: center;
+}
+
+header {
+  width: 95%;
+  margin-left: auto;
+  margin-right: auto;
+  height: auto;
+  padding-top: 2%;
+  padding-bottom: 2%;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  background-color: rgb(253, 236, 239);
+
+  border: 3px darkblue solid;
+}
+
+header img {
+  width: 50px;
+  height: 50px;
+
+  border: 1px red solid;
+}
+
+header div {
+  width: 60%;
+  display: table;
+
+  border: 1px red solid;
+}
+
+header div h4 {
+  width: auto;
+  text-align: right;
+  opacity: 0.6;
+  font-style: italic;
+  margin: 0;
+
+  border: 1px red solid;
+}
+header nav {
+  text-align: right;
+
+  border: 1px red solid;
+}
+
+#welcome {
+  width: 95%;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  border: darkred 3px solid;
+}
+
+#welcome .welcome-text {
+  width: auto;
+  height: 50%;
+  object-fit: contain;
+
+  display: block;
+  object-fit: cover;
+  justify-content: flex-start;
+  padding: 5%;
+  border: 1px red solid;
+}
+
+#welcome .welcome-image {
+  width: 100%;
+  height: 50%;
+  object-fit: cover;
+
+  display: block;
+  border: 1px red solid;
+}
+
+img {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: cover;
+}
+
+#cakes {
+  width: 95%;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+
+  border: darkred 3px solid;
+}
+
+#cakes .cake-div {
+  width: 45%;
+  height: 45%;
+  margin: 2%;
+  object-fit: cover;
+
+  position: relative;
+
+  display: block;
+  border: 1px red solid;
+}
+
+#cakes .cake-div:after {
+  content: "";
+  display: block;
+  padding-bottom: 100%;
+}
+
+#cakes .cake-div img {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+footer {
+  width: 95%;
+  margin-left: auto;
+  margin-right: auto;
+  height: auto;
+
+  background-color: rgb(253, 236, 239);
+  border: 3px darkblue solid;
+}
+
+footer .footer-divs {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+
+  padding: 2%;
+
+  border: 1px red solid;
+}
+
+footer .footer-divs .first,
+.second,
+.third {
+  width: 35%;
+  height: auto;
+  margin: 2%;
+
+  background-color: rgb(214, 214, 214);
+  border: 1px red solid;
+}
+
+footer .copyright {
+  opacity: 0.6;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+  height: auto;
+
+  background-color: rgb(214, 214, 214);
+  border: 1px red solid;
+  text-align: center;
+}
```

Comment:
Add some media queries to make the layout change on desktop!

Reply:
I missed my last push before and I pushed last version which is responsive.